### PR TITLE
A slight bit of crucial maintenance

### PR DESCRIPTION
**Pull request by DandelionSprout**
Greetings, Dandelion Sprout here, the maintainer of *Dandelion Sprout's Nordic Filters* and a general adblocking enthusiast.

Having learned about AdNauseam yesterday, I decided to make this pull request to see how much this extension was being worked on, and whether or not the project intended to strive for functional equivalence with uBlock Origin. What it does, is:

* Synchronised the regional lists with those of uBO.
* * Most crucially, `Finnish Addition to Easylist` should've been removed half a year ago, as it was first ousted from major adblockers due to the maintainer making blocking rules against labour union sites as a whole all of a sudden, followed by him removing all entries in his list.
* Fixed the Chrome download link in the readme.
* Changed some links in the readme from HTTP to HTTPS.
* Removed the CONTRIBUTING.md file, as it quite obviously didn't apply to the AdNauseam repo.

### CODE REVIEW

File: assets/assets.json [hunk 1]
```diff
@@ -531,29 +521,41 @@
...
 		"group": "regions",
 		"off": true,
-		"title": "POL: polskie filtry do Adblocka i uBlocka",
+		"title": "POL: Oficjalne Polskie Filtry do AdBlocka, uBlocka Origin i AdGuarda",
 		"lang": "pl",
 		"contentURL": "https://raw.githubusercontent.com/MajkiIT/polish-ads-filter/master/polish-adblock-filters/adblock.txt",
 		"supportURL": "https://github.com/MajkiIT/polish-ads-filter/issues",
 		"instructionURL": "https://github.com/MajkiIT/polish-ads-filter#polish-filters-for-adblock-ublock-origin--adguard"
 	},
+	"POL-2": {
```
Comment (krystian3w) [inline assets/assets.json]: Good!

**Comment by dhowe** [comment]
Thanks @DandelionSprout. We are indeed still working actively on ADN and would love to get it in sync with UBO, though a) we are quite strapped for dev resources, and b) we stay generally a few releases behind UBO. We are about to release a new version of ADN, based on UBO v1.17.0, which should include your PR. But any addtl thoughts on how better to stay in sync with the various list maintainers would be great.

**Comment by DandelionSprout** [comment]
>based on UBO v1.17.0, which should include your PR.

Nope! Various changes have been done to the uBO version of `assets.json` after 1.17.0, including the removal of `Finnish Addition to EasyList`. After all, I based this pull request on AdNauseam's `assets.json` file as it appeared after the `Merge branch 'upstream1.17.0' into merge1.17.0`-titled commit. From my occasional involvement in uBO, I can also tell that any changes done to the `assets.json` file would go live to all users of all version numbers within minutes, as it is version-independent.

I also see no reason to not update the GitHub repo's readme links either.

**Comment by dhowe** [comment]
Sorry, I meant that once I check and merge this PR, it will be included in our next release.

**Comment by DandelionSprout** [comment]
...Aaaaah. That makes more sense. I too am sorry for originally misreading you.

**Comment by DandelionSprout** [comment]
As for how to get in touch with maintainers, a handful of maintainers and uBO team members can be found at https://github.com/DandelionSprout/adfilt/issues/7, and there's been some very recent small attempts at establishing a forum at https://hub.filterlists.com/ as well.

**Pull request merged**